.github/workflows/deploy-sdk.yml

```diff
@@ -1,6 +1,13 @@
 name: Deploy SDK
 
 on:
+  workflow_dispatch: # manually run this workflow. This allows you to manually deploy things like cococapods, not manually create a git tag. The tag needs to already be created to run this. 
+    inputs:
+      existing-git-tag:
+        description: 'Type name of existing git tag (example: 1.0.3) to checkout and manually deploy'
+        required: true
+        type: string 
+
   push:
     branches: [beta, main, v1] # all branches where deployments currently occur. Make sure this list matches list of branches in  `.releaserc` file.
```

Comment:
do we still need other branches other than `main` ?

Reply:
We will need `v1` listed. We could just remove `beta` from this list.